scripts/THALES_migration/stakingEscrowMigration/executeStakingAndEscrowMigration.js

```diff
@@ -65,14 +65,14 @@ async function executeStakingAndEscrowMigration() {
 			console.log('Skipping ' + migratedStakerOrEscrower.wallet + ' as its a contract!');
 			continue;
 		}
+
 		//send directly if not a staker
 		console.log('Processing migratedStakerOrEscrower ' + migratedStakerOrEscrower);
-		if (migratedStakerOrEscrower.totalStaked == 0 || migratedStakerOrEscrower.unstaking) {
-			let escrowed = Big(migratedStakerOrEscrower.totalEscrowed / 1e18);
-			console.log('Escrowed is ' + escrowed);
-			let unstakingAmount = Big(migratedStakerOrEscrower.unstakingAmount / 1e18);
-			let totalAmount = escrowed.add(unstakingAmount);
-			await thales.transfer(migratedStakerOrEscrower.wallet, w3utils.toWei(totalAmount.toString()));
+		if (migratedStakerOrEscrower.totalStaked == 0) {
+			await thales.transfer(
+				migratedStakerOrEscrower.wallet,
+				w3utils.toWei(migratedStakerOrEscrower.totalEscrowed / 1e18 + '')
```

Comment:
what is the difference between this and 
`(migratedStakerOrEscrower.totalEscrowed / 1e18).toString()`

Reply:
hm, no difference, but this approach is obsolete, I've changed it to 
`		let tx = await thales.transfer(
				migratedStakerOrEscrower.wallet,
				migratedStakerOrEscrower.totalEscrowed + ''
			);`
This ensures to loss of precision